apps/webpack.js

```diff
@@ -464,7 +465,6 @@ function create(options) {
         PISKEL_DEVELOPMENT_MODE: JSON.stringify(piskelDevMode),
         DEBUG_MINIFIED: envConstants.DEBUG_MINIFIED || 0
       }),
-      new webpack.IgnorePlugin({resourceRegExp: /^serialport$/}),
```

Comment:
@madelynkasula could use your thoughts here, we think IgnorePlugin and setting serialport to false as an alias seem like they effectively do the same thing. LMK if you have any information to the contrary!

Reply:
i don't know for sure... they definitely seem similar -- i only see one slight difference, but i don't know what effect that difference has.

setting `alias` to false is documented in the [webpack resolve docs](https://webpack.js.org/configuration/resolve/#resolvealias) as:

> Setting resolve.alias to false will tell webpack to ignore a module.

whereas [`IgnorePlugin`](https://webpack.js.org/plugins/ignore-plugin/) is described as:

> IgnorePlugin prevents the generation of modules for import or require calls matching the regular expressions or filter functions

so basically, is there a functional difference (to us, at least) between ignoring a module and preventing the generation of a module? i'm not totally sure, but i think if we're able to build and run this code and it works, we can be confident that this change is okay

Reply:
Thanks Maddie!

Reply:
> so basically, is there a functional difference (to us, at least) between ignoring a module and preventing the generation of a module? i'm not totally sure, but i think if we're able to build and run this code and it works, we can be confident that this change is okay

For posterity, the functional difference to us is quite hacky -- firmata's import of serialport threw an error when it resolved to `undefined` (when using IgnorePlugin), but not when it resolves to `false` (when using alias). Relevant code here:

https://github.com/firmata/firmata.js/blob/4a3b86b46bedf5d238e19ff9aed08742cc607114/packages/firmata.js/lib/com.js#L53-L63

In previous version, we didn't end up hitting the same error b/c we assigned a value to `com` here:

https://github.com/firmata/firmata.js/blob/v0.16.0/lib/com.js#L39-L40

Reply:
@fisher-alice this does make me wonder -- are you testing just in the browser or in the maker app as well?

Reply:
Oof! Good call! Will check using the Maker app as well.

Reply:
Checked CP Express on Maker App and all components working successfully!